### PR TITLE
docs: Removed iOS 7+

### PR DESCRIPTION
**Pull request by breautek**
### Platforms affected
docs

### Motivation and Context

"Supported platforms iOS 7+" suggested we supported iOS 7 which we don't. We can instead just generically mention iOS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ Set to true to make the statusbar overlay on top of your app. Ensure that you ad
 Supported Platforms
 -------------------
 
-- iOS 7+
+- iOS
 - Android 5+
```
Comment (brodybits) [inline README.md:157]:
```suggestion
- Android
```

**Comment by jcesarmobile** [comment]
I think iOS 7+ can be removed, since it's only specified in one method now, so looks like it should have been removed when it was removed in the other methods, but was missed.

But I wouldn't change the 5+ for Android, since cordova-android 9 is not out yet, and even if it was, the plugin doesn't require cordova-android 9 to work. And also it's used in multiple places, but only removed in one of them.

**Comment by breautek** [comment]
> But I wouldn't change the 5+ for Android, since cordova-android 9 is not out yet, and even if it was, the plugin doesn't require cordova-android 9 to work.

I tend to agree, which is why I didn't originally touched it. @brodybits is it okay I revert this https://github.com/apache/cordova-plugin-statusbar/pull/190/commits/12519643465089630a4395b923cd8a219f7baf13?

**Comment by brodybits** [comment]
> @brodybits is it okay I revert this [1251964](https://github.com/apache/cordova-plugin-statusbar/commit/12519643465089630a4395b923cd8a219f7baf13)?

yes

**Comment by breautek** [comment]
Thanks for the approvals.

The failed check is a disrepency, on travis dashboard, it shows everything is passing https://travis-ci.org/github/apache/cordova-plugin-statusbar (plus this is just a doc change, not a code change anyway).

**Pull request merged**